src/parser/postgresparser.cpp

```diff
@@ -230,7 +230,7 @@ std::unique_ptr<AbstractExpression> PostgresParser::ExprTransform(Node *node) {
       break;
     }
     case T_TypeCast: {
-      expr = TypeCastTransform(reinterpret_cast<TypeCast *>(node));
+      expr = AExprTransform(reinterpret_cast<A_Expr *>(node));
```

Comment:
I don't have a strong opinion on this, but is there a reason we're making it go through AExprTransform first?

Reply:
Yes.. mostly for consistency and a little bit for my pedantic itch.
The original TypeCast expression is T_TypeCast which is handled separately while OPERATOR_CAST is T_A_Expr and those are handled together through AExprTransform.

Reply:
Sure, sounds good.